browser/src/map/handler/Map.SlideShow.js

```diff
@@ -52,37 +52,43 @@ L.Map.SlideShow = L.Handler.extend({
 			return;
 		}
 
-		if (!this._cypressSVGPresentationTest && !this._map['wopi'].DownloadAsPostMessage) {
-			this._slideShow = L.DomUtil.create('iframe', 'leaflet-slideshow', this._map._container);
-			if (this._slideShow.requestFullscreen) {
-				this._slideShow.requestFullscreen();
-			}
-			else if (this._slideShow.msRequestFullscreen) {
-				this._slideShow.msRequestFullscreen();
-			}
-			else if (this._slideShow.mozRequestFullScreen) {
-				this._slideShow.mozRequestFullScreen();
-			}
-			else if (this._slideShow.webkitRequestFullscreen) {
-				this._slideShow.webkitRequestFullscreen();
-			} else {
-				// fallback to "open in new tab"
-				L.DomUtil.remove(this._slideShow);
-				this._slideShow = null;
+		var onPresentation = () => {
```

Comment:
arrow functions are a problem if this is to be ported to 23.05

Reply:
Don't we support the same brower's versions?
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Functions/Arrow_functions#browser_compatibility
What am i missing ?